src/texturePack.ts

```diff
@@ -83,7 +83,6 @@ export const installTexturePack = async (file: File | ArrayBuffer, name = file['
     if (path.endsWith('/')) return
     await mkdirRecursive(dirname(writePath))
     await fs.promises.writeFile(writePath, Buffer.from(await file.async('arraybuffer')))
-    done++
```

Comment:
no way it was that simple, but let's remove ++done in function instead. let's show for others as its better to have side-effect free functions, okay?

Reply:
Yeah it was that simple :)
Roger that, I'll make the change

Reply:
Thank you a lot!